content/docs/guides/schema-diff-tutorial.md

```diff
@@ -0,0 +1,130 @@
+
+1. First, copy your connection string:
+
+    ```bash
+    neonctl connection-string --database-name staging
+    ```
+
+2. Connect to the `staging`` with psql:
+
+    ```bash
+    psql 'postgres://neondb_owner:*********@ep-crimson-frost-a5i6p18z.us-east-2.aws.neon.tech/staging?sslmode=require'
+    ```
+
+2. Create the schema:
+
+    ```sql
+    CREATE TABLE person (
+        id SERIAL PRIMARY KEY,
+        name TEXT NOT NULL,
+        email TEXT UNIQUE NOT NULL
+    );
+    ```
+
+## Step 2: Create your team's branches
+
+If you're still in `psql`, exit using `\q`. Using the CLI, create child branches for each member of your team. Include `--project-id` if you have multiple projects.
+
+```bash
+neonctl branches create --name dev/alex --parent main
+```
+
+```bash
+neonctl branches create --name dev/jordan --parent main
+```
+
+If you want to verity that these branches include the initial schema, connect to that branch and view the `person` table.
+
+1. Here's how to get the connection string for the `staging` database on branch `dev/alex` using the CLI.
+
+    ```bash
+    neonctl connection-string dev/alex --database-name staging
+    ```
+
+    This gives you the connection string which you can then copy.
+
+    ```bash
+    postgres://neondb_owner:*********@ep-hidden-rain-a5pe72oi.us-east-2.aws.neon.tech/staging?sslmode=require
+    ```
+
+1. Connect to `staging` using psql.
+
+    ```bash
+    psql 'postgres://neondb_owner:*********@ep-hidden-rain-a5pe72oi.us-east-2.aws.neon.tech/staging?sslmode=require'
+    ```
+
+1. View the schema for the `person` table we created earlier.
+
+    ```bash
+    \d person
+    ```
+
+    Which shows you the schema:
+
+    ```bash
+    Table "public.person"
+    Column |  Type   | Collation | Nullable |              Default               
+    --------+---------+-----------+----------+------------------------------------
+    id     | integer |           | not null | nextval('person_id_seq'::regclass)
+    name   | text    |           | not null | 
+    email  | text    |           | not null | 
+    Indexes:
+        "person_pkey" PRIMARY KEY, btree (id)
+        "person_email_key" UNIQUE CONSTRAINT, btree (email)
+    ```
+
+You can do the same thing for your `dev/jordan` branch and get identical results.
+
+## Step 3: Update schema on a dev branch
+
+Now, let's make some schema changes on the dedicated branch for `dev\jordan`, adding a new table to store relevant addresses.
+
+This time, we'll use the **SQL editor** in the Neon Console. Make sure you select `dev\jordan` as the branch and `staging` as the database.
+
+Enter this SQL statemenet to create a new `address` table.
+
+```sql
+CREATE TABLE address (
+    id SERIAL PRIMARY KEY,
+    person_id INTEGER NOT NULL,
+    street TEXT NOT NULL,
+    city TEXT NOT NULL,
+    state TEXT NOT NULL,
+    zip_code TEXT NOT NULL,
+    FOREIGN KEY (person_id) REFERENCES person(id)
+);
+```
+
+## Step 4: Visualize the schema differences
```

Comment:
```suggestion
## Step 4: View the schema differences
```

Reply:
We switch to using the UI here because schema diff is only supported via the UI. What do you think about adding tabs for both Console (SQL Editor) and CLI steps? That way the primary flow is UI-based, but the CLI steps are available where they are supported.